bot/data/locales/ru.yml

```diff
@@ -515,13 +519,13 @@ nikogotchi:
       glitched_powerup: "> Сила вашего Никоготчи сильно возрасла после поедания этого недо-оладушка..."
 
       no_food: "Пусто... Вы можете купить еды в магазине!"
-      invalid: Вы не можете скормить это вашему Никоготчи! Пожалуйста обновите!
+      invalid: Вы не можете скормить это вашему Никоготчи! Пожалуйста обновите кнопкой "↻"!
```

Comment:
the refresh emoji is a very good idea, you should change it to `[emoji:refresh]`. i'll make sure to add that before i push to prod